org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/findrefs/ReferenceSearchViewPage.java

```diff
@@ -253,4 +286,12 @@ public TreeViewer getViewer() {
 		return viewer;
 	}
 
+	ReferenceSearchResultContentProvider getContentProvider() {
```

Comment:
why not protected?

Reply:
because package private visibility is enough

Reply:
but is will prevent any customization

Reply:
@szarnekow what do you think?

Reply:
OK, will change